QueryBuilder/FilteredQuery.cs

```diff
@@ -145,6 +145,20 @@ public TQuery WhereNotIn<TModel>(string propertyName, string[] values, string al
             return (TQuery)this;
         }
 
+        /// <summary>
+        /// Add a where isOfModel operator.
+        /// </summary>
+        /// <param name="alias">Optional - Model Alias.</param>
+        /// <returns>ADT query instance.</returns>
+        public TQuery WhereIsOfModel<TFromModel, TWithModel>(string alias = null)
+            where TFromModel : BasicDigitalTwin
+            where TWithModel : BasicDigitalTwin
+        {
+            whereClause.AddCondition(CreateWhereIsOfModelCondition<TFromModel, TWithModel>(typeof(TFromModel), typeof(TWithModel), alias));
```

Comment:
I feel like the added `typeof(TFromModel), typeof(TWithModel)` arguments here are redundant - can't this method derive those type objects from the type generics?

Reply:
Yes, it is possible to do so. Will change that in the next PR.